### PR TITLE
Update user with 3 new props in beta

### PR DESCRIPTION
**Pull request by dkershaw10**

### CODE REVIEW

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/user.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/userProp_djk/api-reference/beta/resources/user.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811082151462106-3363%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FQWd24F%2FUG5yfQQgrZFUBAsh%2B9IY5zrctmnGRAJPgNM%3D&st=2018-11-08T21%3A48%3A48Z&se=2018-12-09T21%3A53%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/user.md [hunk 1]
```diff
@@ -105,6 +107,7 @@ This resource supports:
 |onPremisesSecurityIdentifier|String|Contains the on-premises security identifier (SID) for the user that was synchronized from on-premises to the cloud. Read-only.|
 |onPremisesSyncEnabled|Boolean| **true** if this object is synced from an on-premises directory; **false** if this object was originally synced from an on-premises directory but is no longer synced; **null** if this object has never been synced from an on-premises directory (default). Read-only |
 |onPremisesUserPrincipalName|String| Contains the on-premises `userPrincipalName` synchronized from the on-premises directory. The property is only populated for customers who are synchronizing their on-premises directory to Azure Active Directory via Azure AD Connect. Read-only. |
+|otherMails|String|The fax number of the user.| A list of additional email addresses for the user; for example: `["bob@contoso.com", "Robert@fabrikam.com"]`. Supports $filter|
```
Comment (eketo-msft) [inline api-reference/beta/resources/user.md]: Copy pasta bug?

Comment (eketo-msft) [reply api-reference/beta/resources/user.md]: You fixed already :)

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/user.md](https://github.com/dkershaw10/microsoft-graph-docs/blob/userProp_djk/api-reference/beta/resources/user.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201811082154416967-3363%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LqLbVa8oxdF9Dy%2Bu77JQ6bKakTDscgre%2FI74ZWn%2BGWg%3D&st=2018-11-08T21%3A51%3A30Z&se=2018-12-09T21%3A56%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dkershaw10** [comment]
@lleonard-msft Can you merge this please if you think this is OK.  Eric approved.

**Pull request merged**